### PR TITLE
SUBS-373 MG Update Payment Method

### PR DESCRIPTION
**Pull request by eddieferrer**
* Adds ability to modify MG payment method.

Screen Shot 2020-08-24 at 3 10 34 PM
Screen Shot 2020-08-24 at 3 05 04 PM

**Behavior**

For users who are not opted in to experiment:
Current behavior, minor styling updates on form

Users opted in:

- If user has a BT vaulted card, they will see current payment method and link to update payment method which will take them to the drop in screen to update. On clicking confirm a new MG subscription will be created with new payment method. 

- If user has legacy paypal agreement. They will not see current payment method details, but will see update payment method link. On clicking this link, and going through the drop in interface, they will cancel their paypal auto deposit and create a new one with braintree. 

~~If a user's card expires and they're MG becomes inactive, they will see the same interface, as if they were active. If they are taken to the drop in via the updatePayment=true link, presumably the braintree drop in will handle creating a new MG subscription with a valid payment method.~~

Possible improvements:
- Change behavior to modify autodeposit instead of recreate
- Display messaging or indicator for inactive/expired, etc subscriptions?

### CODE REVIEW

File: src/pages/Autolending/AutolendingWho.vue [hunk 1]
```diff
@@ -304,6 +304,7 @@ export default {
 	.advanced-options-toggle {
 		color: $kiva-textlink;
 		font-weight: 300;
+		padding: 0.5rem;
```
Comment (eddieferrer) [inline src/pages/Autolending/AutolendingWho.vue:307]: Unrelated change, this just makes the toggle link look better when focus style is active

File: src/components/Forms/MonthlyGoodUpdateForm.vue [hunk 2]
```diff
@@ -46,102 +46,108 @@
...
-					v-model="form.category"
-					class="group-dropdown"
-				>
-					<option
-						v-for="(option, index) in lendingCategories"
-						:value="option.value"
-						:key="index"
+			<div class="row align-middle">
+				<div class="column">
+					<strong>Your contribution will:</strong>
```
Comment (eddieferrer) [inline src/components/Forms/MonthlyGoodUpdateForm.vue:135]: No real changes to this component except some alignment changes, and this copy change, see PR screenshot

File: src/pages/Subscriptions/SubscriptionsMonthlyGood.vue [hunk 3]
```diff
@@ -134,9 +249,27 @@ export default {
...
+			}) || {};
+
+			// if experiment and feature flag are BOTH on, show UI
+			const braintreeDropInFeatureFlag = _get(data, 'general.braintreeDropInFeature.value') === 'true' || false;
+			this.showDropInPaymentUpdate = braintreeDropInFeatureFlag && braintreeDropInExp.version === 'shown';
 		},
 	},
+	mounted() {
+		// Check route params (string) for payment method reset case
+		if (this.$route.query.updatePayment === 'true' && this.showDropInPaymentUpdate) {
```
Comment (mcstover) [inline src/pages/Subscriptions/SubscriptionsMonthlyGood.vue:268]: this won't be used to begin with and will require some additional work to enable, but it's ok to keep in here for now.
